src/uwtools/resources/jsonschema/esg-grid.jsonschema

```diff
@@ -0,0 +1,119 @@
+{
+  "allOf": [
+    {
+      "if": {
+        "not": {
+          "required": [
+            "namelist.base_file"
+          ]
+        }
+      },
+      "then": {
+        "properties": {
+          "namelist": {
+            "properties": {
+              "update": {
```

Comment:
Should this be `update_values`? If so, I'm guessing that we need schema unit tests to back up what the schema says here.